test/run-syntax-checks.sh

```
@@ -1538,4 +1540,78 @@ fi
+###############################################################################
+if [ "$requested_cxx_standard" =  "26" ] || [ "$requested_cxx_standard" = "All" ]; then
+SetCxxStandard "26"
+
+if [ "$compiler_enabled" = "gcc" ] || [ "$compiler_enabled" = "All compilers" ]; then
+SetConfigurationName "STL"
+compiler=$gcc_compiler
+PrintHeader
+rm -rdf bgcc
+rm -rdf bclang
+cmake -E make_directory bgcc bclang
+CC=gcc CXX=g++ cmake -E chdir bgcc cmake -DNO_STL=OFF -DETL_USE_TYPE_TRAITS_BUILTINS=OFF -DETL_USER_DEFINED_TYPE_TRAITS=OFF -DETL_FORCE_TEST_CPP03_IMPLEMENTATION=OFF -DETL_CXX_STANDARD=$cxx_standard ..
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)